its/src/test/java/com/sonar/it/shared/ScannerCliTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * SonarSource :: C# :: ITs :: Plugin
+ * Copyright (C) 2011-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.shared;
+
+import com.sonar.orchestrator.Orchestrator;
+import com.sonar.orchestrator.build.BuildResult;
+import com.sonar.orchestrator.build.SonarScanner;
+import com.sonar.orchestrator.container.Edition;
+import com.sonar.orchestrator.locator.MavenLocation;
+import java.io.File;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Regression tests for scanning projects with the scanner-cli.
+ *
+ * Note that this uses a different orchestrator instance than {@link com.sonar.it.csharp.Tests} or {@link com.sonar.it.vbnet.Tests}
+ */
+public class ScannerCliTest {
+  private static final String RAZOR_PAGES_PROJECT = "WebApplication";
+
+  @ClassRule
+  public static final Orchestrator ORCHESTRATOR = Orchestrator.builderEnv()
+    .setSonarVersion(TestUtils.replaceLtsVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE")))
+    .addPlugin(TestUtils.getPluginLocation("sonar-csharp-plugin"))
+    .addPlugin(TestUtils.getPluginLocation("sonar-vbnet-plugin"))
+    // Fixed version for the HTML plugin as we don't want to have failures in case of changes there.
+    .addPlugin(MavenLocation.of("org.sonarsource.html", "sonar-html-plugin", "3.2.0.2082"))
+    .setEdition(Edition.DEVELOPER)
+    .activateLicense()
+    .build();
+
+  @Before
+  public void init(){
+    TestUtils.deleteLocalCache();
```

Comment:
Other tests uses `TestUtils.reset(ORCHESTRATOR);`. Why it is not needed here?

Reply:
because the orchestrator is only used in this test, so it's "clean".
The other tests re-use the same SQ instance.